### DISCUSSION TITLE
ability to favorite stuff

### DISCUSSION THREAD

**Discussion by chirrupted** [justchokingaround/lobster#121] [Ideas]
dont know if this alr exists but a favorite option would be pretty nice (i.e favorite shows to be able to access them with a command like "lobster -f" instead of having to search them up)

**Comment by justchokingaround** [comment]
why not use the history feature for this use case?

**Comment by chirrupted** [comment]
didnt know that existed lol sorry

**Reply by justchokingaround** [reply]
np, you can read more about it here:
https://github.com/justchokingaround/lobster#-c----continue-argument
